com.unity.render-pipelines.high-definition/Editor/Core/TextureCombiner/TextureCombiner.cs

```diff
@@ -241,7 +204,7 @@ public Texture2D Combine(string savePath)
             combinerMaterial.SetVector("_BRemap", m_remapings[2]);
             combinerMaterial.SetVector("_ARemap", m_remapings[3]);
 
-            RenderTexture combinedRT = new RenderTexture(xMin, yMin, 0, RenderTextureFormat.ARGBFloat, RenderTextureReadWrite.sRGB);
+            RenderTexture combinedRT = new RenderTexture(xMin, yMin, 0, GraphicsFormat.R32G32B32A32_SFloat);
```

Comment:
So this is losing the sRGB qualifier here, can we do something or the format just doesn't have an equivalent?

Reply:
In this kind of cases I assumed the sRGB flag was a mistake as float formats are always linear anyway.

Reply:
I wondered if before we did some weird conversion internally, but yes I guess we are good like this